app/src/main/res/values-sw600dp-land/dimens.xml

```diff
@@ -468,6 +468,24 @@
   <dimen name="hints_and_solution_recycler_view_padding_top">4dp</dimen>
   <dimen name="hints_and_solution_recycler_view_padding_bottom">112dp</dimen>
 
+  <!-- Hints summary  -->
+  <dimen name="hints_summary_constraint_layout_layout_margin_start">176dp</dimen>
+  <dimen name="hints_summary_constraint_layout_layout_margin_end">176dp</dimen>
+  <dimen name="hints_summary_constraint_layout_layout_margin_bottom">4dp</dimen>
+
```

Comment:
Here and everywhere else where you have given extra space in dimens.xml remove it as mentioned above